src/state_manager/mod.rs

```diff
@@ -1297,6 +1298,15 @@ where
                             ?actual_receipt,
                             "state mismatch"
                         );
+                        // TODO: Download expected state from network when not available in local DB
```

Comment:
This function requires the state to be present in the database. Validating tipsets received from the network (as opposed to a snapshot file) happens elsewhere. We can remove this TODO.

Reply:
Done.